webhook_server/tests/test_custom_check_runs.py

```
@@ -0,0 +1,860 @@
+    @pytest.fixture
+    def mock_github_webhook(self) -> Mock:
+        """Create a mock GithubWebhook instance with custom checks configured."""
+        mock_webhook = Mock()
+        mock_webhook.hook_data = {
+            "action": "opened",
+            "pull_request": {"number": 123},
+        }
+        mock_webhook.logger = Mock()
+        mock_webhook.log_prefix = "[TEST]"
+        mock_webhook.repository = Mock()
+        mock_webhook.clone_repo_dir = "/tmp/test-repo"
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/test-repo"

(S108)